packages/storybook/stories/CallWithChatComposite/snippets/CallWithChat.snippet.tsx

```diff
@@ -37,14 +38,11 @@ export const CallWithChatExperience = (props: CallWithChatExampleProps): JSX.Ele
     endpoint: props.endpointUrl
   });
 
-  if (adapter) {
-    const options = { callControls: props.callWithChatControlOptions };
-    return <CallWithChatComposite adapter={adapter} fluentTheme={props.fluentTheme} options={options} />;
-  }
-
-  if (credential === undefined) {
```

Comment:
The explicit error handling for `credential` is needed to provide user feedback for a very common error: Providing a corrupt token. This happens most easily when the user swaps the userId and token copying in.

After your PR, adding a corrupt token gives no feedback: The canvas stays stuck in:
![image](https://user-images.githubusercontent.com/82062616/157504058-362aee4a-fc9d-48cf-9f03-931e8f2394a6.png)

and there is this cryptic error in the console:
![image](https://user-images.githubusercontent.com/82062616/157504030-0bcfb7b5-9767-4658-8967-5d7ad01eea52.png)

Reply:
Hmm, that sort of error handling keeps adding that extra complexity where a thrown error should more than suffice.
Agree its a problem in our storybook though, what if I add it to the internal Storybook preview code, but keep the snippet without it?

Reply:
Updated to have a checks for credential -- in a seperate snippet file that just the story preview uses. The snippet remains using the simplified, well commented snippet